services/tasks-svc/internal/patient/api/grpc_test.go

```diff
@@ -21,7 +22,9 @@ import (
 func server(ctx context.Context) (pb.PatientServiceClient, func()) {
 	// Build gRPC service
 	aggregateStore := hwes_test.NewAggregateStore()
-	patientGrpcService := api.NewPatientGrpcService(aggregateStore)
+	patientHandlers := handlers.NewPatientHandlers(aggregateStore)
```

Comment:
if patientHandlers is not used anywhere else, why not move it into the NewPatientGrpcService constructor?